astro/src/pages/services/escape-room/index.astro

```diff
@@ -248,88 +257,54 @@ import { Icon } from "astro-icon/components";
     <div
       class="d-flex flex-column flex-lg-row-reverse gap-4 justify-content-lg-between align-items-center align-items-lg-start"
     >
-      <div class="col-3 col-lg-2">
-        <Image src={escapeRoomLogo} alt="" class="logo" aria-hidden="true" />
+      <div class="col col-lg-4">
+        <ImageFigure
+          src={brailleBlocks}
+          alt="Alphabet blocks with braille"
```

Comment:
I would almost say you don't need alt text since you have a caption (and it can get verbose). Could maybe do a aria-describedby although I haven't used that with a caption before